plugins/youtube_dl_button.py

```diff
@@ -109,6 +109,21 @@
     tmp_directory_for_each_user = Config.DOWNLOAD_LOCATION + "/" + str(update.from_user.id)
     if not os.path.isdir(tmp_directory_for_each_user):
         os.makedirs(tmp_directory_for_each_user)
+    if len(custom_file_name) > 64:
+        await bot.edit_message_text(
+            text="File Name is greater than 64 characters. Reducing File name to 64 charterers.",
+            chat_id=update.message.chat.id,
+            message_id=update.message.message_id
+        )
+        file_exten = custom_file_name.split(".")
```

Comment:
can you add rsplit here? 

consider a situation where file name has multiple . DOTs? 😅